lib/beaker/module_install_helper.rb

```diff
@@ -15,24 +15,25 @@ def install_module
   # the source on the local machine
   def install_module_on(host)
     copy_module_to(host,
-                   source:      @module_source_dir,
+                   source:      $module_source_dir,
                    module_name: module_name_from_metadata)
   end
 
   # This method calls the install_module_dependencies_on method for each
   # host which is a master, or if no master is present, on all agent nodes.
   def install_module_dependencies
-    hosts_to_install_module_on.each do |host|
-      install_module_dependencies_on(host)
-    end
+    install_module_dependencies_on(host)
   end
 
   # This method will install the module under tests module dependencies on the
   # specified host from the dependencies list in metadata.json
   def install_module_dependencies_on(host)
+    host = [host] if host.is_a?(Hash)
```

Comment:
If you change the meaning of the `host` param, you want to pluralize it, and update the docs.